Sources/GameMP/Computer.cpp

```diff
@@ -246,7 +246,7 @@ void LastUnreadMessage(void)
 // go to next/previous message
 void PrevMessage(void)
 {
-  if (_iActiveMessage<_acmMessages.Count()==0) {
+  if (_iActiveMessage < _acmMessages.Count()==0) { // FIXME: DG: what is this about?
```

Comment:
So written in a non-obfuscated way, that would be `if (_iActiveMessage >= _acmMessages.Count())`, right?